src/v2/Components/ArtworkFilter/ArtworkFilters/SizeFilter.tsx

```diff
@@ -63,37 +86,51 @@ const convertRangeToInches = (range: CustomRange) => {
   ]
 }
 
-const convertSizeToInches = (size: CustomSize) => {
-  return {
-    height: convertRangeToInches(size.height),
-    width: convertRangeToInches(size.width),
-  }
+const getValue = (value: CustomRange[number]) => {
+  return value === "*" || value === 0 ? "" : value
 }
 
-const mapSizeToRange = (size: CustomSize) => {
-  return {
-    height: size.height?.join("-"),
-    width: size.width?.join("-"),
+export const getPredefinedSizesByMetric = (metric: Metric) => {
+  if (metric === "cm") {
+    return SIZES_IN_CENTIMETERS
   }
-}
 
-const getValue = (value: CustomRange[number]) => {
-  return value === "*" || value === 0 ? "" : value
+  return SIZES_IN_INCHES
 }
 
-export interface SizeFilterProps {
-  expanded?: boolean
+export const getUnitLabelByMetric = (metric: Metric) => {
+  return metric === "cm" ? "cm" : "in"
 }
 
-type Mode = "resting" | "done"
+export const getCustomSizeRangesInInches = (
+  customSize: CustomSize,
+  sourceMetric: Metric
+) => {
+  let sizes = customSize
+
+  if (sourceMetric === "cm") {
+    sizes = {
+      width: convertRangeToInches(customSize.width),
+      height: convertRangeToInches(customSize.height),
```

Comment:
**suggestion**: Noticed some very long converted values in the query params when selecting custom ranges. Should we use `Math.round` in `convertRangeToInches` as well?

Reply:
Yes, we can do this, but there may be problems with the reverse conversion. For example:
* user specified `100 cm`
* convert from `cm` to `in` (`100cm` -> `39.37007874015748 in`)
* Apply `Math.round` (get `39 in`)
* convert from `in` to `cm` (`39 in` -> `99 cm`)

Reply:
How about rounding to two decimal places?

```typescript
const cm = 100
const inches = Math.round((cm / 2.54) * 100) / 100 // => 39.37
Math.round((inches * 2.54) * 100) / 100 // => 100
```

Reply:
I also considered this option, but there is still a problem with rounding

```javascript
const cm = 1
const inches = Math.round((cm / 2.54) * 100) / 100 // => 0.39
Math.round((inches * 2.54) * 100) / 100 // => 0.99
```

Reply:
But this option will work (when converting from `in` to `cm` and rounding occurs to an integer)

```javascript
const cm = 1
const inches = Math.round((cm / 2.54) * 100) / 100 // => 0.39
Math.round(inches * 2.54) // 1
```

Reply:
Looks good to me 👍🏼

Reply:
I remembered [some details](https://github.com/artsy/force/pull/9096) about why the custom values were not rounded on Force. If we return the rounding to Force, then we will need to add changes to Eigen as well.

Reply:
Ahhh, I see. Uff...

In that case let's skip the rounding and keep the long conversions. They'll only be visible in the URL query params anyway. Shouldn't be too noticeable.

But I think we should also prioritize pushing some of these concerns back from the UI, whether that's in [Metaphysics](https://github.com/artsy/metaphysics) or [Gravity](https://github.com/artsy/gravity) 🔒. From the UI's perspective, it should just request filtering in centimeters and let the backend handle _how_ that happens.

Reply:
> But I think we should also prioritize pushing some of these concerns back from the UI, whether that's in [Metaphysics](https://github.com/artsy/metaphysics) or [Gravity](https://github.com/artsy/gravity) 🔒. From the UI's perspective, it should just request filtering in centimeters and let the backend handle how that happens.

Good suggestion!